packages/fxa-auth-server/scripts/write-emails-to-disk.js

```diff
@@ -121,7 +117,7 @@ function sendMail(mailer, messageToSend) {
     planIntervalCount: 4,
     playStoreLink: 'https://example.com/play-store',
     invoiceNumber: '8675309',
-    cardType: 'mastercard',
+    cardType: 'MasterCard',
```

Comment:
I have no idea why this ever passed. I got some failures telling me it should be `MasterCard` instead of `mastercard`.